### PR TITLE
Including Customer use-case

### PR DESCRIPTION
**Pull request by vrohit13**

### CODE REVIEW

**Review by bcole — COMMENTED** [review]
Looks great overall.

I like the top section (endpoint to hit, request headers to use, mention of OAuth flow) - I wonder if we could standardize (or link to a standard place) for all use cases to have (not just Customer).

Maybe there's a link to the common stuff (endpoints, etc), and each use case just mentions the required scopes (which is specific per use case)?

No need to change it in this PR, we can revisit, it's just my current thoughts.

File: docs/use-cases/customer.md [hunk 1]
```diff
@@ -0,0 +1,373 @@
+---
+layout: default
+title: Customer
+nav_order: 5
```
[the remark this answers is not in the document]
Comment (vrohit13) [reply docs/use-cases/customer.md:4]: Ah. Sure. I realized this once the other PRs were merged.

File: docs/use-cases/customer.md [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+
+## Customer
+
+### API URL
+
+The Customer API allows creation and read operations, and is exposed against a new endpoint. Make a POST request to the endpoint below.
+
+**Endpoint:**
+
+Prod: [https://public.api.intuit.com/2020-04/graphql](https://public.api.intuit.com/2020-04/graphql)
```
[the remark this answers is not in the document]
Comment (vrohit13) [reply docs/use-cases/customer.md:16]: I agree.

File: docs/use-cases/customer.md [hunk 3]
```diff
@@ -0,0 +1,373 @@
...
+
+
+#### Query by ID
+
+Query:
+
+```
+query fetchCustomer($id: String!){
+    company {
+    customers (filter: {id: {equals: $id}}) {
```
[the remark this answers is not in the document]
Comment (vrohit13) [reply docs/use-cases/customer.md:178]: Tabs vs Spaces (We can't have both :-D). I will fix them.

File: docs/use-cases/customer.md [hunk 4]
```diff
@@ -0,0 +1,373 @@
...
+
+#### Query by Name
+
+
+Query:
+
+```
+query fetchCustomerByName($displayName: String!){
+    company {
+    id
```
[the remark this answers is not in the document]
Comment (vrohit13) [reply docs/use-cases/customer.md]: Fixed them.

**Comment by vrohit13** [comment]
> Looks great overall.
> 
> I like the top section (endpoint to hit, request headers to use, mention of OAuth flow) - I wonder if we could standardize (or link to a standard place) for all use cases to have (not just Customer).
> 
> Maybe there's a link to the common stuff (endpoints, etc), and each use case just mentions the required scopes (which is specific per use case)?
> 
> No need to change it in this PR, we can revisit, it's just my current thoughts.

Yes. I feel we could have this information in a separate page and have a hyperlink from all other pages. We could take this up later.

**Review by bcole — APPROVED** [review]

**Pull request merged**